core/src/main/java/com/linecorp/armeria/client/endpoint/StaticEndpointGroup.java

```diff
@@ -26,7 +26,10 @@
 
 /**
  * A static immutable {@link EndpointGroup}.
+ *
+ * @deprecated use {@link EndpointGroup}
```

Comment:
- use -> Use
- missing period
- `{@link EndpointGroup#of(EndpointGroup...)}` ?

Reply:
I guess you forgot this :-)

Reply:
Oops